src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --button-color: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408vh;
```

Comment:
The height property seems to be set to '408vh' which is likely a typo or incorrect value. The viewport height (vh) unit is relative to the height of the viewport, so '408vh' means the element will be much taller than the viewport. Please check if this is the intended behavior and consider using a more appropriate unit or value.